### PR TITLE
chore: Use webpack to bundle extension

### PR DESCRIPTION
**Pull request by CsCherrYY**
`vscode-extension-telemetry-wrapper` depends on `vscode-extension-telemetry`, which has some dependencies like `diagnostic-channel-publishers` and `applicationinsights`, both of them have some requires in **try-catch** block, which are unresolved in rollup, see https://github.com/rollup/plugins/issues/496.

So we try webpack here.

### CODE REVIEW

File: extension/webpack.config.js [hunk 1]
```diff
@@ -25,6 +25,9 @@ const config = {
     'applicationinsights-native-metrics':
       'commonjs applicationinsights-native-metrics',
     'diagnostic-channel-publishers': 'commonjs diagnostic-channel-publishers',
+    '@grpc/proto-loader': '@grpc/proto-loader',
```
Comment (Eskibear) [inline extension/webpack.config.js:28]: do we need to append "commonjs "?

Comment (CsCherrYY) [reply extension/webpack.config.js:28]: regarding https://github.com/microsoft/vscode-gradle/blob/6c55dcdc188952863176d368970949214023ed7f/extension/rollup.config.js#L35

I think we should append `commonjs` for the last two modules.

Comment (Eskibear) [reply extension/webpack.config.js:28]: ok

**Review by Eskibear — APPROVED** [review]
LGTM now

**Pull request merged**